### DISCUSSION TITLE
v0.20.4

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1509] [Releases]
> [!WARNING]  
> `v0.20.x` has breaking changes and requires action. Read the original release notes here: https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0

## What's Changed
* [Bug] Use table columns not attributes on results table by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1500
* [Feature] Made checking for internet connection optional by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1507
* Release v0.20.4 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1508

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.20.3...v0.20.4

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.4'>v0.20.4</a>.</em>

**Comment by bigrichwood** [comment]
Still broken for me, will log a new issue.

**Reply by alexjustesen** [reply]
Make sure to drop `SPEEDTEST_PING_URL` from your env vars or set it to `false`, this will disable the check.

**Reply by bigrichwood** [reply]
SPEEDTEST_PING_URL is not defined in my compose file, and pings are not blocked anywhere. 

See the issue you've just closed for details.

**Reply by alexjustesen** [reply]
Linking to #1510 to lineage

**Comment by seh2000** [comment]
@alexjustesen 
Hi Alex,
I have been away for some days, but here tonight I managed to update to v0.20.4 :)
However I have questions to two of the environment variables, marked with **?** in the below on what am I to write as I am not sure.
I presume the rest are OK - or?
Many thanks in advance!
Steen 

            - SPEEDTEST_SCHEDULE=*/30 * * * *
            - SPEEDTEST_SERVERS=4604 #Dublin (Blacknight)
            - PRUNE_RESULTS_OLDER_THAN=60
            - CHART_DATETIME_FORMAT= **?**
            - DATETIME_FORMAT= **?**
            - DISPLAY_TIMEZONE=Europe/Dublin
            - APP_TIMEZONE=Europe/Dublin``

**Reply by alexjustesen** [reply]
I added a link to the docs just now but you can use https://www.php.net/manual/en/datetime.format.php for chart and datetime formatting.

**Reply by seh2000** [reply]
Hi Alex,
Thanks - got it to work nicely (for me that is).

<img width="299" alt="image" src="https://github.com/alexjustesen/speedtest-tracker/assets/74658646/c117f13c-d058-4cef-a17e-8d3981d81545">

<img width="79" alt="image" src="https://github.com/alexjustesen/speedtest-tracker/assets/74658646/f3ffe406-4563-4828-9898-25e29959cf0f">

**Comment by CPSibo** [comment]
Just updated from v0.19 and looks like everything's working smoothly.

Thank you for your continued work on this 🎉
